src/docs/sphinx/dev_guide/misc_tasks.rst

```diff
@@ -49,6 +49,72 @@ Questions we need to answer include:
 .. note :: Pull in content from ../web/build_system/thirdparty_deps.rst ...
            fill in gaps and make sure it it up-to-date...
 
+Updating a TPL to a new version
+-------------------------------
+
+Follow these steps to update Axom to use a new released version of one
+of the TPL libraries.
+
+Create a branch to work on the task.
+
+Update the `packages.yaml` files in `scripts/uberenv/spack_configs`.
+Find the entry for the library you wish to update and change the
+version number. Do this for every `packages.yaml` for every system type,
+ncluding those in the `docker` subdirectory.
+
+Update `scripts/uberenv/packages/[package name]/package.py` for the
+library you are changing. Usually this can be done by copying directly
+from `var/spack/repos/builtin/packages/[package name]/package.py` in
+the Spack distribution. Clone the repo at `github.com/spack/spack` to
+find this file and copy it. Verify that the new version you intend to
+use is included in this file in the list of `version()` entries.
+
+Do a test installation in your local directories
+
+   $ scripts/llnl/build_tpl.py -d ../your/local/install/path
+
+This will do a full installation of the TPLs, which you can check to verify
+that the correct new version is installed. Also, it produces new host-config
+files that you should use to build and test Axom with this installation. If
+any changes to Axom code are needed to work with the new TPL update, make
+these changes and test them.
+
+When you are confident that everything is correct, log in as user
+`atk` to each of the machines named in Axom's standard host-configs and run
+
+   $ scripts/llnl/build_tpl.py -d /usr/WS1/axom/libs
```

Comment:
I think you can just run `scripts/llnl/build_tpl.py`

But you should run this from an allocation since the script does not do this for you.